grasp_generation/utils/isaac_validator.py

```diff
@@ -1060,7 +1165,13 @@ def _get_camera_intrinsics(self) -> Tuple[float, float, float, float]:
         assert math.isclose(cx, cy) and math.isclose(cx, 0) and math.isclose(cy, 0)
         return fx, fy, cx, cy
 
-    def _create_one_split(self, split_name: str, split_range: np.ndarray, folder: str):
+    def _create_one_split(
+        self,
+        split_name: str,
```

Comment:
What happens with the split_name now? Does this change the path or anything?

If it works, then I'm happy :)

Reply:
the current behavior is that the `split_name` changes the path of where the images are stored. For instance, now, the directory of the images is `images`. The other change is that `transforms.json` no longer has the `split_name` appended to it.